package login.proto;

import "ResultCodes.proto";
import "CharSummary.proto";

message GetCharSummaryListReply {
 required ResultCode result_code = 1;
 repeated CharSummary summary_list = 2;
}

